spec/models/video_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe Video do
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
+
+  describe "search_by_title" do
```

Comment:
There is a convention while building describe messages in RSpec - you should prepend to the method name `.` in case of a class method and `#` in case of an instance method. Read more here - http://www.betterspecs.org/#describe